api/Controllers/pageController.js

```diff
@@ -0,0 +1,72 @@
+import { retrievePage, storePage } from "../Utils/fileUtils.js";
+import { HttpStatusCodes } from "../Utils/httpStatusCodes.js";
+
+export const getPage = async (req, res) => {
+    var query = 'SELECT * from space_pages_view WHERE organization_name = $1 AND space_name = $2 AND folder_name = $3 AND page_name = $4';
```

Comment:
It is generally preferred to use let or const (use const everywhere you aren't reassigning a variable) instead of var